sql/core/src/test/resources/sql-tests/inputs/except-all.sql

```diff
@@ -0,0 +1,146 @@
+CREATE TEMPORARY VIEW tab1 AS SELECT * FROM VALUES
```

Comment:
I feel it's ok to move these tests to `except.sql`.

Reply:
@maropu I thought we like to keep these sql files relatively small and not contain too many sqls.